### PR TITLE
init commit for static home page

### PR DESCRIPTION
**Pull request by FrancescoWard**
Hello team,
I hope this finds you well.
Please review for integration.
Thank you.

Francesco

### CODE REVIEW

**Review by mcontado — APPROVED** [review]

**Review by mcontado — CHANGES_REQUESTED** [review]
Hi Francesco, please pull the latest from Development branch into your local: ```git pull origin Development```  and then do ```git push``` again.

**Review by mviencek — CHANGES_REQUESTED** [review]
Head is missing title element.  Unclosed div at line 74.  Please fix.

**Comment by FrancescoWard** [comment]
Hi Team,
Thanks for reviewing.
Resolved unclosed div on line #74 + minor modifications. Not perfect but better than before.
Is more reviewing required or can I merge ?
Thanks,
Francesco

**Review by mviencek — APPROVED** [review]
youre still missing the <title> element from the head.  Please add for your next commit.  also just an fyi, the map to calandar to map flow doesnt work.  clicking back and forth between the two tabs, the map doesn't reappear.

File: views/styles/styles.css [hunk 1]
```diff
@@ -203,6 +203,8 @@ a.navbar-brand{
     margin-right: 78%;
 }
 
+<<<<<<< HEAD
```
Comment (mcontado) [inline views/styles/styles.css]: please fix this merge conflicts

File: views/styles/styles.css [hunk 2]
```diff
@@ -224,4 +226,5 @@ body {
     width: 550px;
     height: 400px;
     background-color: grey;
-}
\ No newline at end of file
+}
+>>>>>>> 4a8417e09bf83ecab30323f08a6729d6e2e5551f
```
Comment (mcontado) [inline views/styles/styles.css]: please remove this and fix merge conflicts

**Comment by FrancescoWard** [comment]
Hi Team,
Thanks for the additional review.
Please see latest. 
Francesco

**Review by mcontado — APPROVED** [review]

**Pull request merged**